src/main/scala/sbtdocker/DockerPush.scala

```diff
@@ -44,5 +47,20 @@ object DockerPush {
     val process = Process(command)
     val exitValue = process ! processLog
     if (exitValue != 0) sys.error("Failed to push")
+
+    val PushedImageId = ".* sha256:([0-9a-f]+) .*".r
```

Comment:
Since it is the image digest that is matched, the name of the constant should reflect that.

Reply:
yes!